src/ParallelEnv.f90

```diff
@@ -1052,7 +1052,7 @@ SUBROUTINE gather_str2D_MPI_ENV_type(myPE,sendbuf,root)
     ENDDO
   ENDDO
   !This is somewhat arbitrary, but giving preference to lowest rank's data
-  CALL myPE%allReduceMinI(SIZE(charProc),charProc)
+  CALL myPE%allReduceMaxI(SIZE(charProc),charProc)
```

Comment:
This change actually reverses the above comment. Like the comment says its some what arbitrary, but it had the effect of always preserving the data on the master process. I don't know that one way is better than the other, but the comment should at least be conssitent.

Reply:
oh, yeah, I didn't even read the comment.  I'll update that.

Reply:
fixed